context.go

```diff
@@ -285,6 +289,36 @@ func NewContext() *Context {
 	return ctx
 }
 
+// searches for Blueprints files under baseDir
+func findBlueprintsFromDir(baseDir string) (filePaths []string, err error) {
```

Comment:
If we're going to implement this here, can we also export a list of dependencies so that the dep file actually works?

Reply:
this is removed in the next patchset at your suggestion